### PR TITLE
[#5m8rgj] Dask saturn

### PR DESCRIPTION
**Pull request by jsignell**
Replaces the current CLI with a per-cluster tornado server that can be used to `adapt` and `scale` a cluster.

This PR depends on the corresponding PR in saturn proper (atlas).

### CODE REVIEW

**Comment by jsignell** [comment]
## To test

Build this image:
```
cd images/daskdev-sat
eval $(minikube docker-env)
docker build -t "saturncluster" .
```

Change `saturn/.env_deps` to point to it: `DASKDEV_SAT_IMAGE=saturncluster`
Add port forwarding to the Makefile:

```make
k8s-daskdev-pf:
	kubectl -n main-namespace port-forward svc/admin-myjupyter-dask 8788
```

In Atlas (as admin user), spin up a jupyter called myjupyter
Then start a dask

Edit the k8s services template
```
kubectl edit services -n main-namespace admin-myjupyter-dask 
# add a section for the app to forward 8788
```

**Comment by jsignell** [comment]
I think this one is probably fine to go in sometime soon (definitely pre-https://github.com/saturncloud/saturn/pull/220). I am not sure if I should change the version or something for the image.

File: daskdev-sat/Dockerfile [hunk 1]
```diff
@@ -9,13 +9,12 @@ RUN apt-get -qq update && \
     rm -rf /var/lib/apt/lists/*
 ENV SHELL /bin/bash
 
-RUN conda install -y dask=2.8.1  distributed
+RUN conda install -y -c conda-forge distributed tornado kubernetes dask-kubernetes python-kubernetes
 RUN conda install -y -c https://conda.saturncloud.io/pkgs sutils=0.5.1
-RUN conda install -y -c conda-forge kubernetes=1.13.5 dask-kubernetes=0.10.0 python-kubernetes=9.0
```
Comment (forana) [inline daskdev-sat/Dockerfile:14]: I think we may actually want to keep the versions called out, here - if we don't, that'll mean that upstream changes could get pulled in without us being aware of them until users complain. On the flip side, though, I don't think it's really a big issue - thoughts @hhuuggoo ?

Comment (jsignell) [reply daskdev-sat/Dockerfile:14]: Yeah we probably want to pin. I was just floating these to update them.

**Pull request merged**